src/DNSSECStack.ts

```diff
@@ -20,18 +20,23 @@ export class DNSSECStack extends Stack {
      */
   constructor(scope: Construct, id: string, props: DNSSECStackProps) {
     super(scope, id, props);
-    this.setDNSSEC();
+    this.setDNSSEC(props);
   }
 
-  setDNSSEC() {
+  setDNSSEC(props: DNSSECStackProps) {
 
     const parameters = new RemoteParameters(this, 'params', {
       path: `${Statics.ssmZonePath}/`,
       region: 'eu-west-1',
     });
     const zoneId = parameters.get(Statics.ssmZoneIdNew);
 
-    const accountDnssecKmsKeyArn = SSM.StringParameter.valueForStringParameter(this, Statics.ssmAccountDnsSecKmsKey);
+    var paramName = Statics.ssmAccountDnsSecKmsKey;
+    if (props.branch === 'production') {
+      // TODO moved temprarely to another parameter so that a new KMS key can be created
+      paramName += '/moving';
```

Comment:
Deze haal je op uit parameter store? Daar zie ik 'm nog niet bestaan, klopt dat?

Reply:
Deze parameter bestaat alleen in auth-prod en wordt aangemaakt vanuit de dns-management repository (in us-east-1). 
Dit is nodig om auth-prod aan te kunnen maken op een uniforme manier vanuit dns-management.

Reply:
Check, ik had alleen in eu-west-1 gekeken, dan snap ik 'm!